[docs] mark expo-firebase-* as deprecated (#19687)
expo/expo: docs/pages/versions/unversioned/sdk/firebase-analytics.mdx

import { Terminal } from '~/ui/components/Snippet';
import { InlineCode } from '~/components/base/code';

> > **warning** **Deprecated.** This module will be removed in SDK 48. Check out our [Migration Guide](https://expo.fyi/rnfs-migration-guide) for help migrating to [React Native Firebase](https://rnfirebase.io/).

Reviewer: ```suggestion
> **warning** **Deprecated:** This module will be removed in SDK 48. Check out our [Migration Guide](https://expo.fyi/rnfs-migration-guide) for help migrating to [React Native Firebase](https://rnfirebase.io/).
```

I'd suggest using `:` instead of `.` after the word Deprecated. It's not exactly a convention, just a small nit because of the pattern we follow across all docs when putting up a deprecation notice 😄

> > **warning** **Deprecated:** This module will be removed in SDK 48. Check out our [Migration Guide](https://expo.fyi/firebase-migration-guide) for help migrating to [React Native Firebase](https://rnfirebase.io/).

Reviewer 2: ```suggestion
> **warning** **Deprecated:** This module will no longer be available in SDK 48. [Learn how to migrate to React Native Firebase](https://expo.fyi/firebase-migration-guide).
```
